src/App.tsx

```diff
@@ -1,24 +1,106 @@
-import React, { FC } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { FC, useState } from 'react';
+import {
+  Box,
+  Button,
+  Container,
+  CssBaseline,
+  TextField,
+} from '@material-ui/core';
+import Todo from './components/Todo/Todo';
+import { Todo as ITodo } from './components/Todo/typings';
 
-const App: FC = () => (
-  <div className="App">
-    <header className="App-header">
-      <img src={logo} className="App-logo" alt="logo" />
-      <p>
-        Edit <code>src/App.tsx</code> and save to reload.
-      </p>
-      <a
-        className="App-link"
-        href="https://reactjs.org"
-        target="_blank"
-        rel="noopener noreferrer"
+const App: FC = () => {
+  const [todoList, setTodoList] = useState<ITodo[]>([]);
+  const [newTodoName, setNewTodoName] = useState<string>('');
+  const [inputTodoError, setInputTodoError] = useState<string>('');
+
+  const changeTodoNameHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setNewTodoName(event.target.value);
+  }
```

Comment:
I suggest changing `event` to `evt` to KISS. Also using sugar syntax in one line.

```suggestion
  import { ChangeEvent } from 'react';
  ...
  const changeTodoNameHandler = (evt: ChangeEvent<HTMLInputElement>) => 
    setNewTodoName(evt.target.value);
```

Reply:
Resolved on ac40b7bd5b1418375c16876812c614ac7cb7f273